tests/test_indicators.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+import sys
+sys.path.insert(0, '/Users/brynne/Python/Documents/GitHub/Crypto_Bin/bin_strat')
+import indicators
+
+import pandas as pd 
+from pandas._testing import assert_frame_equal
+import numpy as np
+
+
+
+
+def test_macd_calc():
+	"""
+	This is to test that the output DataFrame from _MACD_calc function provides the expected calculation by comparing the output to a stand-alone .csv file 'price_history_macd_test.csv'
+	inputs: price_history is a dummy .csv file with made up Time, Close, and Volume
+	? Cannot import _macd_calc to test this??
```

Comment:
Good find 😝  To your point, private functions don't usually need to be tested because they will be called in public functions like `indicators.macd_signal` (or so the conventional logic goes). I see why you'd want to test this though - we can either move this function to another module, make it public, or just skip the tests. Leave that up to you, not mad either way

Reply:
After manually calculating the formulas last night, i realized how well it tests the results of the full function as it stands now. Defo will need to test more complicated formulas in the future but since this is just MACD, i think the tests currently are sufficient.

Reply:
just hype of the `assert_frame_equal()` function ;)

Reply:
Sounds good 👍